### PR TITLE
add navbar

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

I added navbar

## Screenshots

NAVBAR

## Trello Card

https://trello.com/c/hjsmNcal/31-navbar

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -7,7 +7,11 @@
     <meta name="author" content="Wolox" />
     <link rel="stylesheet" type="text/css" href="application.css" />
   </head>
-  <body>
+  <body class="container-books">
+    <div class="navbar">
```
Comment (Dorian30) [inline maquetado/index.html]: I would use `<nav>` tag instead

File: maquetado/index.html [hunk 2]
```diff
@@ -7,7 +7,11 @@
...
+      <img class="wolox-image" src="image/LogoWolox.png" alt="Book Image" />
+      <p class="logout">Logout</p>
```
Comment (Dorian30) [inline maquetado/index.html]: We usually use `<a>` tags for buttons

File: maquetado/scss/detail.scss [hunk 3]
```diff
@@ -35,9 +40,37 @@
...
+  width: 200px;
+}
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  border-top: 5px solid $light-blue;
+  box-shadow: 5px 5px 10px 0 $black-05;
+  display: flex;
+  flex-direction: row;
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: There is no need to set this as row. By default it works like that once you add `display: flex`

File: maquetado/scss/detail.scss [hunk 4]
```diff
@@ -35,9 +40,37 @@
...
+  height: 70px;
+  justify-content: space-between;
+  padding: 0 100px 0 100px;
```
Comment (Dorian30) [inline maquetado/scss/detail.scss]: Use the shorthand `padding: top right bottom left` 

```suggestion
  padding: 0 100px;
```

**Review by Dorian30 — CHANGES_REQUESTED** [review]

File: maquetado/scss/colors.scss [hunk 5]
```diff
@@ -2,5 +2,7 @@
 $green: #bed23c;
 $white: #fff;
 $grey: #828282;
+$light-blue: #00adee;
+$light-grey: #f4f4f4;
```
Comment (gabrielizalo) [inline maquetado/scss/colors.scss:7]: Remember sort attributes alphabetically...

File: maquetado/scss/detail.scss [hunk 6]
```diff
@@ -35,9 +40,36 @@
...
+  align-items: center;
+  background-color: $white;
+  border-top: 5px solid $light-blue;
+  box-shadow: 5px 5px 10px 0 $black-05;
+  display: flex;
+  height: 70px;
+  justify-content: space-between;
+  padding: 0 100px;
+  position: fixed;
+  margin-bottom: 3px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss]: Can `3px` be `5px`?

File: maquetado/scss/detail.scss [hunk 7]
```diff
@@ -35,9 +40,36 @@
...
+  width: 100%;
+}
+
+.logout {
+  color: $black;
+  font-family: Roboto;
+  font-size: 18px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss]: Can `18px` be `20px`?

File: maquetado/scss/detail.scss [hunk 8]
```diff
@@ -35,9 +40,36 @@
...
+  line-height: 21px;
```
Comment (gabrielizalo) [inline maquetado/scss/detail.scss]: Can `21px` be `20px`?

**Review by gabrielizalo — APPROVED** [review]

**Review by Dorian30 — APPROVED** [review]

**Pull request merged**